CONTRIBUTING.md

```diff
@@ -52,19 +51,9 @@ The full list of dependencies and other StreetComplete-related projects is liste
 
 ### Suggesting new quests
 
-We appreciate all quest ideas you and others have, but some ideas cannot be added into the app due to various design decisions and requirements. This is required in order to make a great app for all users of StreetComplete, whose contributions actually help the OpenStreetMap project. A quest which nobody can solve and where users potentially choose wrong options adding wrong data to OpenStreetMap hurts not only OSM, but also this app.
+Not all ideas for quests are actually eligible to be included in this app. So, before you suggest a new quest, it is very important that you read the [Quest Guidelines for StreetComplete](https://github.com/westnordost/StreetComplete/wiki/Adding-new-Quests-to-StreetComplete).
 
-What you need to do:
-
-1. Search whether the quest you want to suggest is [already implemented in StreetComplete](https://wiki.openstreetmap.org/wiki/StreetComplete/Quests) or [already suggested in the issue tracker](https://github.com/westnordost/StreetComplete/issues).
```

Comment:
I would not remove that.

It is quite important users search whether the quest has been suggested already and stuff like that. Also here they had a step-by-step guide, which more users may actually follow than some running text.

Reply:
It is important, but it is important for *any* issue one creates in *any* project to not create duplicates. It should go without saying. People who create some duplicates after all are usually the kind of people who would also not read a CONTRIBUTING.md or the stuff in the issue template. We've seen it often enough that users just left everything as-is in the template and we have seen it often enough that people did not read the quest contribution guidelines.